graql/language/rule-validation.feature

```diff
@@ -631,3 +631,72 @@ Feature: Graql Rule Validation
       };
       """
     Then the integrity is validated
+
+
+  Scenario: a rule can use a super-relations's role in the sub-relation.
+    Given graql define
+      """
+      define
+
+      person
+      plays president,
+      has hair;
+
+      hair sub attribute,
+      value string;
+
+      citizenship sub relation,
+      relates citizen;
+
+      presidency sub citizenship,
+      relates president as citizen;
+
+      orange-man sub rule,
+      when {
+          (citizen: $a) isa presidency;
+      }, then {
+          $a has hair "orange";
+      };
+      """
+    Given the integrity is validated
+    When get answers of graql query
+      """
+      match $x sub rule; get;
+      """
+    When concept identifiers are
+      |     | check | value                   |
+      | ORA | label | orange-man              |
+      | RUL | label | rule                    |
+    Then uniquely identify answer concepts
+      | x   |
+      | ORA |
+      | RUL |
+
+  Scenario: When defining a rule with a variable atom type, an error is thrown.
```

Comment:
These scenario names should be updated to follow the same conventions as the other scenarios: all lowercase and no full stop at the end